src/web/core/src/main/java/org/geoserver/web/GeoServerHomePage.java

```diff
@@ -75,7 +75,7 @@ public GeoServerHomePage() {
             String contactEmail = contact.getContactEmail();
             HashMap<String, String>params = new HashMap<String, String>();
             params.put("version", version);
-            params.put("contactEmail", contactEmail);
+            params.put("contactEmail", (contactEmail == null ? "geoserver@example.org" : contactEmail));
```

Comment:
Kind of wish we could just turn the link off if no email address has been provided.